assets/cluster-monitoring-operator/prometheus-rule.yaml

```diff
@@ -46,6 +46,8 @@ spec:
       record: cluster:container_spec_cpu_shares:ratio
     - expr: sum(rate(container_cpu_usage_seconds_total{container="",pod!=""}[5m])) / sum(machine_cpu_cores)
       record: cluster:container_cpu_usage:ratio
+    - expr: sum(irate(container_network_receive_bytes_total{pod!=""}[5m])) BY (pod, interface) + on(pod, interface) group_left(network_name) pod_network_name_info
```

Comment:
tested on 4.16.0-0.nightly-2024-04-08-024331 with the metrics expr, there is "many-to-many matching not allowed: matching labels must be unique on one side" error
```
$ token=`oc create token prometheus-k8s -n openshift-monitoring`
$ oc -n openshift-monitoring exec -c prometheus prometheus-k8s-0 -- curl -k -H "Authorization: Bearer $token" 'https://thanos-querier.openshift-monitoring.svc:9091/api/v1/query?' --data-urlencode 'query=sum(irate(container_network_receive_bytes_total{pod!=""}[5m])) BY (pod, interface) + on(pod, interface) group_left(network_name) pod_network_name_info' | jq
{
  "status": "error",
  "errorType": "execution",
  "error": "found duplicate series for the match group {interface=\"eth0\", pod=\"installer-6-qe-daily-416-0411-gxzbk-master-0\"} on the right hand-side of the operation: [{__name__=\"pod_network_name_info\", container=\"kube-rbac-proxy\", endpoint=\"metrics\", instance=\"10.128.0.4:8443\", interface=\"eth0\", job=\"network-metrics-service\", namespace=\"openshift-kube-scheduler\", network_name=\"ovn-kubernetes\", pod=\"installer-6-qe-daily-416-0411-gxzbk-master-0\", prometheus=\"openshift-monitoring/k8s\", service=\"network-metrics-service\"}, {__name__=\"pod_network_name_info\", container=\"kube-rbac-proxy\", endpoint=\"metrics\", instance=\"10.128.0.4:8443\", interface=\"eth0\", job=\"network-metrics-service\", namespace=\"openshift-kube-controller-manager\", network_name=\"ovn-kubernetes\", pod=\"installer-6-qe-daily-416-0411-gxzbk-master-0\", prometheus=\"openshift-monitoring/k8s\", service=\"network-metrics-service\"}];many-to-many matching not allowed: matching labels must be unique on one side"
}
```